Tests/Indicators/IndicatorExtensionsTests.cs

```diff
@@ -35,9 +35,9 @@ public void PipesDataUsingOfFromFirstToSecond()
             second.Of(first);
 
             var data1 = new IndicatorDataPoint(DateTime.UtcNow, 1m);
-            var data2 = new IndicatorDataPoint(DateTime.UtcNow, 2m);
-            var data3 = new IndicatorDataPoint(DateTime.UtcNow, 3m);
-            var data4 = new IndicatorDataPoint(DateTime.UtcNow, 4m);
+            var data2 = new IndicatorDataPoint(DateTime.UtcNow.AddSeconds(1), 2m);
+            var data3 = new IndicatorDataPoint(DateTime.UtcNow.AddSeconds(2), 3m);
+            var data4 = new IndicatorDataPoint(DateTime.UtcNow.AddSeconds(3), 4m);
```

Comment:
Looks like all these indicator tests should Not be using `DateTime.UtcNow` or `DateTime.Today` really but a fixed reference time